01-creating-a-transaction/index.js

```diff
@@ -7,29 +7,33 @@ const sdk = new SDK({ url: 'https://eos.greymass.com' }) // loaded with default
 
 async function main() {
     const session = await sdk.login('myapp') // login also restores existing session unlike anchor-link, can have option for forcing re-login
-    
+
     // Loads contract by name
     const contract = await sdk.getContract('rewards.gm')
-    
+
     // Work with table data
-    const usersTable = await contract.getTable('users')
-    const row = await usersTable.getByPrimary('teamgreymass')
-    const amount = row.getColumn('amount')
-    
+    const usersTable = contract.getTable('users')
```

Comment:
I'm assuming this wouldn't just load the table rows automatically - so this doesn't need to await?

This would return a `Table` object, which then could be queried.

Reply:
Maybe we need some sort of different naming convention rather than `.getTHING`. 

If it's calling an API, maybe it's `.fetchByPrimary` or `.fetch`, where as if it's just returning a `Table` object it uses `.get`?